contrib/pom.xml

```diff
@@ -678,5 +678,10 @@
       <version>1.3.0</version>
       <optional>true</optional>
     </dependency>
+    <dependency>
```

Comment:
This doesn't belong here. Not everyone using contrib wants to pull in the Avro module.